src/css/variables.module.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["../variables.module.scss","variables.module.css"],"names":[],"mappings":"AAeA;EACE,mBAHa;EAIb,gBALW;ACTb","file":"variables.module.css"}
```

Comment:
why do you have map files committed into repo?

Reply:
I didn't know I have. Is it something bad?